{{ cookiecutter.repo_name }}/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '2'
+services:
+    luigid:
+      image: axiom/docker-luigi
+      container_name: luigid
+      ports:
+          - "8082:8082"
+
+      command: luigid
+      networks:
+          - dev
+
+    dask-scheduler:
+      image: drtools/drtools:latest
+      command: dask-scheduler
+      container_name: dask-scheduler
+      networks:
+        - dev
+      depends_on:
+          - dask-worker
+      ports:
+        - "8787:8787"
+        - "8788:8788"
+
+
+    dask-worker:
+      image: drtools/drtools:latest
+      volumes:
+        - "../data:/home/drtools/{{ cookiecutter.repo_name }}/data"
+        - "../models:/home/drtools/{{ cookiecutter.repo_name }}/models"
```

Comment:
Why did you choose to map these 2 directories?

Reply:
I've changed this now to map root_dir into the container. This will give us persistency and worker needs it to read and write data.